docs/source/references/api_spec.md

```diff
@@ -242,6 +242,7 @@ Also returns current status of peer in json string:
     "view_changes": 0
 }
 ```
+- __CAUTION__: Almost all fields are 64-bit integers and should be handled with care in JavaScript
```

Comment:
can we be more specific about which fields are 64-bit int?

Reply:
Fixed to encourage readers to refer actual `Status` struct, but since the struct is defined nested, also added details here